src/RequestHandle.cpp

```diff
@@ -480,4 +562,83 @@ auto curl_write_data(
     return data_length;
 }
 
+auto curl_xfer_info_old(
+    void* pointer_to_request,
+    double, double , double , double
+) -> int
+{
+    auto* raw_request_ptr = static_cast<RequestHandle*>(pointer_to_request);
+
```

Comment:
return raw_request_ptr->hasDownloadedRequiredBytes() -> int;  This function is duplicated with its sister function

Reply:
Realized today that we don't need these callbacks to abort the CURL request, so I've removed them. More in main comment.